src/main/java/ladder/domain/Floor.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Floor {
+
+    private static final int FIRST_LINE = 0;
+    private static final int INDEX_DIFFERENCE = 1;
+    private final List<Line> lines = new ArrayList<>();
+
+
+    public Floor(int lineSize) {
+        for (int i = 0; i < lineSize; i++) {
+            lines.add(new Line());
+        }
+    }
+
+    public List<Line> getLines() {
+        return lines;
+    }
+
+    public void makeFloor(List<Integer> lineValues) {
+        for (int i = 0; i < lines.size(); i++) {
+            makeLineAt(i, lineValues.get(i));
+        }
+    }
+
+    private void makeLineAt(int index, int value) {
+        if (isMakeAble(index)) {
+            lines.get(index).make(LineSource.of(value));
+        }
+    }
+
+    private boolean isMakeAble(int index) {
+        if (index == FIRST_LINE) {
+            return true;
+        }
+
+        int previousIndex = index - INDEX_DIFFERENCE;
+
+        return !lines.get(previousIndex).isExist();
```

Comment:
부정문은 코드 가독성을 떨어뜨린답니다 ~ 
그래서 코드는 되도록 긍정문으로 한번에 알아볼 수 있도록 작성하시길 추천드려요 ㅎㅎ 
```suggestion
        return lines.get(previousIndex).notExist();
```

Reply:
감사합니다 ㅎㅎ
가독성 측면말고도 해당 함수가 isExist로 받은 값의 부정값을 구한다는 측면에서 기능이 난잡해 진다는 면도 단점이 될 수 있나요??

Reply:
기능이 난잡해졌다는 뜻은 성능이 저하된다는 뜻일까요? ㅎㅎ 
그렇다면 이 부분이 유의미한 성능저하를 일으키지는 않을 것 같아요 ~

Reply:
해당 기능에 !연산이 하나 더하니까 이 함수가 담당하는 기능이 하나 더 는 것같아서 난잡해진다는 표현을 사용한 것 같습니다!ㅎㅎ